packages/vkui/src/components/UsersStack/UsersStack.module.css

```
@@ -21,7 +25,7 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  color: var(--vkui--color_text_contrast);
+  color: var(--vkui--color_text_contrast_themed);
```

В макетах неверный цвет, правильный просто contrast

```suggestion
  color: var(--vkui--color_text_contrast);
```